gaphor/ui/modelmerge/tests/test_organize_uml.py

```diff
@@ -182,7 +182,7 @@ def test_applied_stereotype(element_factory, modeling_language, create):
     apply_stereotype(class_item.subject, stereotype)
 
     current = ElementFactory()
-    all(compare(current, element_factory))
+    all(compare(current, current, element_factory))
```

Comment:
For my understanding, are these tests now comparing the current against itself?

Reply:
They conpare current against element_factory. The resulting changes are created in the current model.

I have to admit, the api is still a bit messy.

Reply:
Thanks for the explanation!